pkg/virt-controller/services/template.go

```diff
@@ -1204,16 +1204,30 @@ func getMemoryOverhead(vmi *v1.VirtualMachineInstance) *resource.Quantity {
 
 	// Add fixed overhead for shared libraries and such
 	// TODO account for the overhead of kubevirt components running in the pod
-	overhead.Add(resource.MustParse("128M"))
+	overhead.Add(resource.MustParse("138Mi"))
 
 	// Add CPU table overhead (8 MiB per vCPU and 8 MiB per IO thread)
 	// overhead per vcpu in MiB
 	coresMemory := resource.MustParse("8Mi")
+	var vcpus int64
 	if domain.CPU != nil {
-		vcpus := hardware.GetNumberOfVCPUs(domain.CPU)
-		value := coresMemory.Value() * vcpus
-		coresMemory = *resource.NewQuantity(value, coresMemory.Format)
+		vcpus = hardware.GetNumberOfVCPUs(domain.CPU)
+	} else {
+		// In case guest CPU topology isn't set, for some reason, take value from resources request or limits.
```

Comment:
Can we add here that this `for some reason` is due to the fact that we did not always set in the mutator? Maybe even add a date. Another option would be a clear unit-tests which explicitly tests this scenario with enough context info given (does one exist?).

Reply:
yeah, unit test (that doesn't involve the mutator) exists. 
I'll add a comment too.